apps/wallet-mobile/src/Nfts/Nfts.tsx

```diff
@@ -21,67 +23,37 @@ export const Nfts = () => {
   const styles = useStyles()
   const {track} = useMetrics()
 
+  const wallet = useSelectedWallet()
+  const balances = usePortfolioBalances({wallet})
+
   // use case: search nfts
   useSearchOnNavBar({
     title: strings.title,
     placeholder: strings.search,
     noBack: true,
   })
+  const {search, isSearching} = useSearch()
 
-  const wallet = useSelectedWallet()
-  const [isManualRefreshing, setIsManualRefreshing] = React.useState(false)
```

Comment:
this was related to this ticket https://emurgo.atlassian.net/browse/YOMO-502

Reply:
no longer needed, all data is synchronously available now.